### DISCUSSION TITLE
Capture groups?

### DISCUSSION THREAD

**Discussion by vedantroy** [pest-parser/pest#549] [Q&A] [answered]
Is there an elegant way to capture a small snippet in a rule without having to create an entire new rule? Example:

```
foo_bar = { "\\baz{" ~ ASCII_ALPHANUMERIC+ ~ "}" ~ ("\r" | "\n") }
```
I want to grab whatever is represented by `ASCII_ALPHANUMERIC+` without creating a whole new rule.

**Comment by CAD97** [comment, marked as answer]
Unfortunately no, the unit of capture is the rule.
